gpio/python/gpioPython/torizonPackages.json

```diff
@@ -1,8 +0,0 @@
-{
-    "deps": [
-        "python3-libgpiod"
```

Comment:
Here remove just the `python3-libgpiod` lines

Reply:
Done. Thanks!